### DISCUSSION TITLE
Disable launchpad

### DISCUSSION THREAD

**Discussion by Shlomixg** [gitkraken/vscode-gitlens#3322] [Q&A] [answered]
Hi,

How do I disable launchpad?
We're not using GitHub and don't need this feature.

**Comment by brunotomaz-dev-stm** [comment]
Hi, I need this too.
I even use GitHub, but since I enabled the launchpad I have been having problems connecting gitlens to github because of the launchpad, so I prefer to disable it.

**Comment by eamodio** [comment, marked as answer]
You can use the `Hide` action here, to "disable" the status bar indicator of launchpad. Launchpad will still work on-demand, but won't refresh in the background when the indicator is hidden.

![image](https://github.com/gitkraken/vscode-gitlens/assets/641685/7574236f-42ea-4431-912a-fbfcbbc32acf)

Another option is to set `gitlens.launchpad.indicator.polling.enabled` to `false` and you can keep the status bar indicator as a trigger, but it won't load data until you open it.

**Reply by piavgh** [reply]
this is correct
